### DISCUSSION TITLE
Commercial Support

### DISCUSSION THREAD

**Discussion by pavlos-christoforou** [pocketbase/pocketbase#1019] [Q&A]
We have come across PocketBase whilst researching low code solutions for the purposes of building proof of concept applications for our (internal) clients. We have decided to include PocketBase in the list of candidates based on its simplicity, good design choices and excellent attention to detail. Well done PocketBase team.

Relative to other solutions, we believe we will need support to be able to use it for our  purposes. Is there commercial support available and if yes can someone point us to the right contact info?

Thanks

**Comment by ganigeorgiev** [comment]
PocketBase is an open source hobby project maintained primarily by me and therefore I cannot provide dedicated commercial support.
If you have questions or difficulties while working with PocketBase, feel free to open an issue/discussion and I'll try to help as best as I can.

With that said, please keep in mind that PocketBase is still under active development (it was released just ~3-4 months ago) and full backward compatibility is not guaranteed until v1.0.0 (you can check our [roadmap](https://github.com/orgs/pocketbase/projects/2)).
An example for a breaking change is the upcoming v0.8.0 release (most likely at the end of this week) that comes with several improvements, but most importantly refactored and simplfied users management per #376 (you can find information about all changes in the [v0.8.0-rc1 changelog](https://github.com/pocketbase/pocketbase/releases/tag/v0.8.0-rc1)).

**Reply by pavlos-christoforou** [reply]
Thank you Gani for the prompt response much appreciated, all clear and understood.

**Comment by benallfree** [comment]
@pavlos-christoforou I don't know if this helps at all, but at https://pockethost.io we are considering adding paid support tiers at some point. For now the service is free, but if managed hosting and support makes sense for your business case, feel free to message me.
